docs-src/python/project-structure.md

```diff
@@ -0,0 +1,88 @@
+---
+id: project-structure
+title: Boilerplate Temporal Application project code
+sidebar_label: Boilerplate project
+description: Discover the minimum code I need to create a boilerplate Temporal Application.
+tags:
+  - python sdk
+  - developer guide
+  - project setup
+---
+
+**What is the minimum code I need to create a boilerplate Temporal Application?**
+
+Let’s start with a single Activity Workflow and register those functions with a Worker.
+
+After we get the Worker running and have started a Workflow Execution, we will add a testing framework.
+
+### Project structure
+
+You can organize Temporal Application code to suit various needs in a way that aligns with the idiomatic style of the language you are working in.
+This includes structuring your files according to your organization's best practices.
+
+However, there are some general ways to think about organizing code.
+
+The best practice is to group Workflows together, Activities together, and separate your Worker process into a standalone file.
+Often this happens respectively per use case, business process, or domain.
+
+For monorepo-style organizational techniques, consider a designated Workflow directory for each use case and place each Workflow in its own file, but also maintain a dedicated place for shared Activities.
+
+For example, your project structure could look like this:
+
+```text
+/monorepo
+    /shared_activities
+        | payment.py
+        | send_email.py
+    /background_check
+        /workflows
+            | background_check_workflow.py
+        /activities
+            | ssn_trace_activity.py
+        /worker
+            | main.py
+    /loan_application
+        /workflows
+            | loan_application_workflow.py
+        /activities
+            | credit_check_activity.py
+        /worker
+            | main.py
+    /tests
+       | pytest.ini
+       | __init__.py
+	   | workflow_tests.py
+       | activity_tests.py
+```
+
+If you are following along with this guide, your project will look like this:
```

Comment:
You are inconsistent with yourself here. The above listing shows `tests` adjacent with `background_check` and this sample shows `tests` inside `backgroundcheck`. Even if the `tests` remain a bit inconsistent which is fine, at the least the underscores in the name should be consistent